dkist/net/client.py

```diff
@@ -98,6 +98,11 @@ def _process_table(results: "DKISTQueryResponseTable") -> "DKISTQueryResponseTab
                 results[colname][none_values] = np.nan
             results[colname] = u.Quantity(results[colname], unit=unit)
 
+        if "Average Fried Parameter" in results.colnames:
+            r_none_values = np.array(results["Average Fried Parameter"] == None)
+            if r_none_values.any():
+                results["Average Fried Parameter"][r_none_values] = np.nan
```

Comment:
Do we also need to change the type of the underlying column in this case? Is it not an object array because of the None's and we want it to be a float?